projects/table/src/lib/cell/generic-cell.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { ContentChild, Directive, TemplateRef } from '@angular/core';
+
+import { NgxAbstractTableCellDirective } from './cell.directive';
+
+@Directive({
+	selector: 'ngx-table-cell',
+	providers: [
+		{
+			provide: NgxAbstractTableCellDirective,
+			useExisting: NgxTableCellDirective,
+		},
+	],
+})
+export class NgxTableCellDirective extends NgxAbstractTableCellDirective {
+	/**
+	 * A template for the header of the cell
+	 */
+	@ContentChild('headerTmpl', { static: false })
+	public headerTemplate: TemplateRef<any>;
```

Comment:
Open vraag. Gebruiken we hier niet beter een `unknown` ipv. een `any`? Eigenlijk weet de component niet wat hier binnen gaat komen 🤔